--- tellico-1.3.5/src/translators/pilotdb/libflatfile/DB.cpp~	2009-11-03 16:11:57.000000000 +0200
+++ tellico-1.3.5/src/translators/pilotdb/libflatfile/DB.cpp	2009-11-03 16:12:07.794304369 +0200
@@ -242,7 +242,7 @@
            kdDebug() << "list view is corrupt" << endl;
         }
         // Determine the length of the name string.
-        pi_char_t* null_ptr = reinterpret_cast<pi_char_t*>
+        const pi_char_t* null_ptr = reinterpret_cast<const pi_char_t*>
             (memchr(chunk.data() + 4, 0, 32));
         if (null_ptr)
             lv.name = std::string((char *) (chunk.data() + 4),
--- tellico-1.3.6/src/cite/actionmanager.cpp~	2008-01-07 07:33:53.000000000 +0200
+++ tellico-1.3.6/src/cite/actionmanager.cpp	2011-01-10 21:46:34.197702109 +0200
@@ -20,7 +20,7 @@
 
 using Tellico::Cite::ActionManager;
 
-ActionManager::ActionManager* ActionManager::self() {
+ActionManager* ActionManager::self() {
   static ActionManager self;
   return &self;
 }
