--- tellico-1.3pre3_orig/src/barcode/barcode_v4l.h     2008-01-12 
06:55:14.000000000 +0100
+++ barcode_v4l.h       2008-01-15 10:43:50.000000000 +0100
@@ -17,6 +17,19 @@
 #define GETELEM(array,index,default) \
        (index < sizeof(array)/sizeof(array[0]) ? array[index] : default)

+/*
+ *  Taken from xawtv-3.5.9 source 8/15/01 by George Staikos <staikos@kde.org>
+ */
+#ifdef __STRICT_ANSI__
+#undef __STRICT_ANSI__
+#define FOO__STRICT_ANSI__
+#endif
+#include <asm/types.h>
+#ifdef FOO__STRICT_ANSI__
+#define __STRICT_ANSI__ 1
+#undef FOO__STRICT_ANSI__
+#endif
+
 //#include <linux/videodev2.h>
 #include <linux/videodev.h>
